Telcodocs 2274 RN: Configuring namespaced SR-IOV resources (#99114)
openshift/openshift-docs: release_notes/ocp-4-20-release-notes.adoc

For more information, see xref:../networking/multiple_networks/secondary_networks/creating-secondary-nwt-other-cni.adoc#nw-multus-bond-cni-object_configuring-additional-network-cni[Configuration for a Bond CNI secondary network].

> [id="ocp-4-20-networking-namespaced--sriov-app-owners_{context}"]

Reviewer: ```suggestion
[id="ocp-4-20-networking-namespaced-sriov-app-owners_{context}"]
```

> ==== Directly manage your SR-IOV Networks in your application namespace

Reviewer: IMO this reads like a command or description. A noun-phrase might be more appropriate for RN headings:

SR-IOV Network Management in Application Namespaces

> This release introduces namespaced SR-IOV networks, allowing application owners to directly create and manage their own SR-IOV networks within their application namespaces. This feature gives you greater control over your network configurations and helps simplify your workflow. 

Reviewer: There's new guidance about release note format here: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes-features-enhancements

I've added a suggest re-write below to align a bit more with the new format. And also removes anthropomorphism: https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism

> This release introduces namespaced SR-IOV networks, allowing application owners to directly create and manage their own SR-IOV networks within their application namespaces. This feature gives you greater control over your network configurations and helps simplify your workflow. 

Reviewer: ```suggestion
Openshift 4.20 introduces namespaced SR-IOV networks. You can use this feature to directly create and manage SR-IOV networks within your application namespaces. Namespaced SR-IOV networks provide greater control over your network configurations and helps to simplify your workflow. 
```

Previously, creating an SR-IOV network required a cluster administrator to configure it for you. Now, you can manage these resources directly in your own namespace, which offers several key benefits:

> * Increased autonomy and control: You can now create your own SriovNetwork objects, removing the need to involve a cluster administrator for network configuration tasks. 

Reviewer: ```suggestion
* Increased autonomy and control: You can now create your own `SriovNetwork` objects, removing the need to involve a cluster administrator for network configuration tasks. 
```

* Enhanced security: Managing resources within your own namespace improves security by providing better separation between applications and helps prevent unintentional misconfigurations.

> * Simplified permissions: This new approach streamlines your permissions and reduces operational overhead.

Reviewer: "This new approach" is a bit ambiguous IMO. Consider stating the feature to remove ambiguity, for example,  "Using namespaced SR-IOV networks streamlines...."
